### ISSUE TITLE
Minor color contrast issues

### ISSUE THREAD

**Issue by erosselli** [djangocon/2024.djangocon.us#35]
I noticed that there's some minor color contrast issues in a couple of sections of the DjangoCon US 2024 website. These can be checked automatically with an extension like [Accessibility Insights for Web ](https://accessibilityinsights.io/docs/web/overview/).

The sections where I found issues are the following:
1. In the "other hotels" section of the [venue page](https://2024.djangocon.us/venue/), there's some blue text on a light blue background that doesn't meet the minimum contrast ratio suggested by WCAG AA guidelines  (this one is the one with the lowest contrast ratio)
2. In the Pre-Conference and Conference Schedule sections of the [about page](https://2024.djangocon.us/about/), the months of the dates are shown in white text over a red background which also doesn't meet the minimum contrast ratio.
3. In the same page, the "Conference Features" text (white text on blue background) also has this problem. This is also an issue in other texts with the same foreground and background colors as this one, like the "A friendly conference for a friendly community" in the [home page](https://2024.djangocon.us/).

Images of where the issues are, as detected by the Accessibility Insights for Web Chrome extension:
<img width="400" alt="Screenshot of the Other Hotels section of the venue page of the website, showing where the contrast issue is" src="https://github.com/djangocon/2024.djangocon.us/assets/67162025/3f5a391a-3d12-44f6-9fe5-bdb25de4173a">
<img width="500" alt="Screenshot of the Pre-Conference and Conference Schedule sections of the website, showing where the contrast issue is" src="https://github.com/djangocon/2024.djangocon.us/assets/67162025/90e6076c-bd32-4c29-9287-71a8f4ffba89">
<img width="500" alt="Screenshot of the Conference Features section of the website, showing where the contrast issue is" src="https://github.com/djangocon/2024.djangocon.us/assets/67162025/b54a8090-5098-4cf1-9027-c06712380f44">

Just wanted to flag these so that we can improve the website accessibility 😄 I'm happy to open a PR to address this.

**Comment by mtrythall** [comment]
Thanks for the catch. I figured some of these might pop up. We'll address these soon as it'll require some design changes.

**Comment by mtrythall** [comment]
All fixed. Thanks for filing.
